setup.py

```diff
@@ -61,7 +61,9 @@ def get_version(*file_paths):
         "django-model-utils>=2.0,<4",
         "faker>=0.7.0,<3.0.0",
     ],
-    extras_require={":python_version<'3'": ["future>=0.16.0"]},
+    extras_require={
+        ":python_version<'3'": ["future>=0.16.0"],
```

Comment:
Can be removed when we don't support <3 anymore

Reply:
Then maybe remove also python 2 from here also:
'Programming Language :: Python :: 2',
'Programming Language :: Python :: 2.7',

Reply:
ah yes, of course! :heavy_check_mark: